searx/external_bang.py

```diff
@@ -0,0 +1,43 @@
+import json
+from os.path import join
+
+from searx import searx_dir
+
+# bangs data coming from the following url convert to json with
+# https://raw.githubusercontent.com/jivesearch/jivesearch/master/bangs/bangs.toml
+# https://pseitz.github.io/toml-to-json-online-converter/
+# NOTE only use the get_bang_url
+
+bangs_data = {}
+with open(join(searx_dir, 'data/bangs.json')) as json_file:
+    for bang in json.load(json_file)['bang']:
+        for trigger in bang["triggers"]:
+            bangs_data[trigger] = {x: y for x, y in bang.items() if x != "triggers"}
+
+
+def get_bang_url(search):
```

Comment:
A tiny thing: search should be renamed search_query to avoid confusion about the type of the object.

Reply:
Oh good point will add that!

Reply:
Fixed it!